console

```diff
@@ -16,6 +20,29 @@ $application = require __DIR__ . '/res/bootstrap.php';
 
 $cli = $application->cli();
 
+$cli->command('serve', function (\Stratify\Http\Application $http, ContainerInterface $container, OutputInterface $output) {
+    $loop = React\EventLoop\Factory::create();
+
+    $db = $container->get(Connection::class);
+
+    $server = new Server(function (ServerRequestInterface $request) use ($http, $db) : ResponseInterface {
+        // Reset the DB connection
+        $db->close();
```

Comment:
I don't know why but in several projects I tried to use this interface but always ended up using `Doctrine\DBAL\Connection` directly instead.